fastai/optimizer.py

```diff
@@ -63,7 +89,12 @@ def _update(state, new=None):
 class Optimizer(_BaseOptimizer):
     "Base optimizer class for the fastai library, updating `params` with `cbs`"
     _keep_on_clear = ['force_train', 'do_wd']
-    def __init__(self, params, cbs, train_bn=True, **defaults):
+    def __init__(self,
+        params:Tensor, # Parameters and hyper parameters
+        cbs, # `Callback` list for the optimizer
```

Comment:
This is not a fastai training loop `Callback`, but rather an Optimizer callback

Reply:
thanks for pointing this out @warner-benjamin!

@tyoc213: `cbs`  is a list of functions that will be composed when applying the step. The type annotation would probably be `(list, callable)` or something like that...

Reply:
@tmabraham mmm from what @warner-benjamin wrote, then I think it should just be `Optimizer` is that right?

Reply:
Changing to `list`